### ISSUE TITLE
After openening score, navigating with Left/Right arrow starts from the title frame instead on notes

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#25371]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

1. Open score
2. Navigate with right arrow - the title frame is selected instead of the first note or rest

### Supporting files, videos and screenshots

https://github.com/user-attachments/assets/98ea79db-b42d-4218-8899-fa89e55a81e1

### What is the latest version of MuseScore Studio where this issue is present?

4.5 nightly

### Regression

Yes, this works in the latest released version and is broken in the nightly builds

### Operating system

all

### Additional context

If you navigate with Alt+Right arrow, the first frame should be selected.

Only navigation without Alt should skip the frame and go straight to the first note or rest.

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by shubham-shinde-442** [comment]
If this isn't too urgent, I'd like to fix this issue.

**Comment by DmitryArefiev** [comment]
This occurs not only in master but in **4.4.4** too (doesn't happen in 4.4.3)
